source/github-helpers/index.ts

```diff
@@ -68,6 +68,16 @@ export const getRepositoryInfo = (repoUrl: string = location.pathname.slice(1)):
 	return {owner, name};
 };
 
+export interface PRRepositoryInfo {
+	user: string;
+	repository: string;
+}
+
+export const getPRRepositoryInfo = (): PRRepositoryInfo => {
```

Comment:
No need to have its own type:

```suggestion
export const getPRRepositoryInfo = (): RepositoryInfo => {
```

Reply:
But I want user/repository over owner/name

Reply:
There's zero reason to return a different type. We can discuss whether it's worth changing `RepositoryInfo` separately.